ee/modules/502-delivery/hack/get_manifests.sh

```diff
@@ -0,0 +1,157 @@
+#!/bin/bash
+#
+# Copyright 2022 Flant JSC
+# Licensed under the Deckhouse Platform Enterprise Edition (EE) license. See https://github.com/deckhouse/deckhouse/blob/main/ee/LICENSE
+#
+#
+#
+###########################################################################
+# CALL THIS SCRIPT FROM THE MODULE DIRECTORY
+#  ./hack/get_manifests.sh
+#
```

Comment:
Please add a brief explanation what this script do and when it should be called.

Reply:
Did it

Reply:
done